iam/package.json

```diff
@@ -16,7 +16,8 @@
     "prod:start": "tsc && node .",
     "test": "jest --verbose",
     "prettier": "prettier --write .",
-    "lint": "tsc --noEmit && eslint --ext .ts,.js,.tsx ."
+    "lint": "tsc --noEmit && eslint --ext .ts,.js,.tsx .",
+    "checkProviders:base-goerli": "npx tsx src/scripts/checkOnChainProvidersAreInSync.ts https://base-goerli.g.alchemy.com/v2/ 0x14a33 0xa652BE6A92c7efbBfEEf6b67eEF10A146AAA8ADc"
   },
```

Comment:
Add commands for other networks as well.
Also update the deployment pipeline to run this commands